doc/source/examples/analysis/polymers_and_membranes/README.rst

```diff
@@ -0,0 +1,14 @@
+.. -*- coding: utf-8 -*-
+
+==========================
+Polymers and membranes
+==========================
+
+MDAnalysis has several analyses specifically for polymers, membranes, and membrane proteins.
+
+.. toctree::
+   :maxdepth: 1
+
+   /examples/analysis/polymers_and_membranes/polymer
+   /examples/analysis/polymers_and_membranes/hole
+   /examples/analysis/polymers_and_membranes/hole2
```

Comment:
The hole2 docs show up with the same title as the hole ones. Perhaps put "Deprecated", "Old", or somthing else in the title of the old module?

Reply:
References in the hole notebooks did not resolve.

Reply:
Hole2 docs still contain reference to HOLEtraj
```
One of the limitations of the hole program is that it can only accept PDB files. In order to use other formats with hole, or to run hole on trajectories, we can use the hole.HOLEtraj class with an MDAnalysis.Universe
```

Reply:
The gif of the hole surface is cute but the gA channel is not properly shown in VMD because of the distortions along the normal mode. To get a nice representation do the following:

```python
import MDAnalysis as mda; from MDAnalysis.tests import datafiles as data

# write out the structure without ETA (which elNemo ignored :-p)
u = mda.Universe(data.PDB_HOLE)
u.select_atoms("not resname ETA").write("elNemo_ga.pdb")
```

Use the above
```
vmd elNemo_ga.pdb path/to/1grm_elNemo_mode7.pdb
```
and delete the first frame in the trajectory.

We need to to remove ETA (the ethanolamine C-terminus) from the original PDB file because the elNemo trajectory does not contain it and so you cannot directly use PDB_HOLE as the "topology" for MULTI_PDB_HOLE.

This should give you a nicer representation of the channel around the pore.

Reply:
Can you use your new `plot_order_parameters(..., aggregrator=...)` function to do the most common application of a hole trajectory profile, the "average HOLE profile" with "mean(R) ± std(R)" (as at the end of https://github.com/MDAnalysis/binder-notebook/blob/master/notebooks/analysis/hole-basics.ipynb )?

Reply:
No, but you could plot the fillbetween standard deviation with on the returned axis from `aggregator=np.mean`, or just make the plot yourself with the data returned from `over_order_parameters`. I could add another plotting function for this?